src/components/profile/edit/EditContact.vue

```diff
@@ -231,6 +246,9 @@ export default {
   background-color: var(--white);
   border: 2px solid var(--gray-20);
   cursor: not-allowed;
+  opacity: 1;
+  color: var(--gray-60);
+  height: calc(100% - 4px); /* account for border */
```

Comment:
`box-sizing: border-box;` should do the same.

Reply:
we use `box-sizing: border-box` globally (as in on `*`). Although it stops the input from getting any wider because of its border, it weirdly can still make the input taller. Have done [a quick codepen](https://codepen.io/hidde/pen/YMGogB)

Reply:
:confounded:  :+1: